app/models/support_interface/ucas_matches_filter.rb

```diff
@@ -54,5 +58,22 @@ def action_needed_filter
         }],
       }
     end
+
+    def action_taken_filter
+      options = UCASMatch.distinct(:action_taken).pluck(:action_taken).uniq.map do |action|
```

Comment:
Do we need `uniq` in this instance, as `distinct` should remove any duplicate entries.

Reply:
For some reason it doesn't quite work. I think I will remove the "No action taken" because it's not playing nicely in the filter and it's not that useful. 

<kbd><img width="167" alt="Screenshot 2020-12-07 at 10 53 37" src="https://user-images.githubusercontent.com/38078064/101342440-77ae2880-387a-11eb-8170-56199f545cf2.png"></kbd>